src/styles/blocks/_catalog.scss

```diff
@@ -14,9 +14,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+
+    transition: transform, color 0.3s;
```

Comment:
you need to set the animation duration for each, right now transform happens instantly if you check the demo